Dockerfile

```diff
@@ -1,9 +1,9 @@
 # syntax = docker/dockerfile:1.1-experimental
 
-ARG RUNC_VERSION=v1.0.0-rc91
-ARG CONTAINERD_VERSION=v1.3.6
-# containerd v1.4 for integration tests
-ARG CONTAINERD_ALT_VERSION=v1.4.0-beta.2
+ARG RUNC_VERSION=v1.0.0-rc92
+ARG CONTAINERD_VERSION=v1.4.0
+# containerd v1.3 for integration tests
+ARG CONTAINERD_ALT_VERSION=v1.3.7
```

Comment:
`BUILDKIT_INTEGRATION_CONTAINERD_EXTRA="containerd-1.4` needs to be changed in this file

Reply:
updated